interview-questions.md

```diff
@@ -8,62 +8,63 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. What is the difference between a parameter and an argument?
 
-  Your answer:
+  Your answer: A parameter is a data type that you use in a function to allow for a dynamic function. An argument is a value that you pass to the parameter to change the value
 
-  Researched answer:
+  Researched answer: Parameters are like variables that can change depending on the inputed value of the argument. Arguments places whatever data types are given to them into the parameter of a function
 
 
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-  Your answer:
+  Your answer: The predefined parameters would be two of the same data types, like two arrays. My understanding is that .map needs to be used in higher order functions whilst utilizing arrays or iterations. I don't know what the optional parameters are but I'll be sure to research the answer.
 
-  Researched answer:
+  Researched answer: Predefined parameters would be functions objects or any primitive data type. An optional parameter would be "this".
 
 
 
 3. What is the difference between map and filter?
 
-  Your answer:
+  Your answer: Map adds two of the same data types together, while filter removes certain values from a data type and creates a new data type with the removed values.
 
-  Researched answer:
+  Researched answer: Map creates a new array by taking the values of two arrays and puts them together to form a new array. Filter goes through each value of an array and takes out certain ones to create a new array.
```

Comment:
.map() is only run on one array at a time and returns an array the same length as the initial array.  This comes in handy when we need to modify each value or display each value.